test/client/getBalances.ts

```diff
@@ -29,66 +29,88 @@ describe("getBalances", function () {
         assertResultMatch(result, responses.getBalances, "getBalances");
       });
 
-      it("getBalances - limit", async function () {
-        const options = { limit: 3, ledgerVersion: 123456 };
-        this.mockRippled.addResponse(
-          "account_info",
-          rippled.account_info.normal
-        );
-        this.mockRippled.addResponse(
-          "account_lines",
-          rippledAccountLines.normal
-        );
-        this.mockRippled.addResponse("ledger", rippled.ledger.normal);
-        const expectedResponse = responses.getBalances.slice(0, 3);
-        const result = await this.client.getBalances(test.address, options);
-        assertResultMatch(result, expectedResponse, "getBalances");
-      });
+      // it("getBalances - limit", async function () {
```

Comment:
Are we going to replace these tests with tests for the new `getBalances` function?

Reply:
Yes!